common/local_filestore.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helper functions for using the local_filestore."""
+
+import os
+
+from common import new_process
+from common import filesystem
+
+
+def cp(  # pylint: disable=invalid-name
+        source,
+        destination,
+        recursive=False,
+        parallel=False):  # pylint: disable=unused-argument
+    """Executes "cp" command from |source| to |destination|."""
+    # Create intermediate folders for `cp` command to behave like `gsutil.cp`.
+    for file_or_dir_path in [source, destination]:
+        dirpath = os.path.dirname(os.path.abspath(file_or_dir_path))
+        if not os.path.exists(dirpath):
+            filesystem.create_directory(dirpath)
+
+    command = ['cp']
+    if recursive:
+        command.append('-r')
```

Comment:
nit: `--recursive` 
in code let's use the non-abbreviated flags when possible to improve readability

Reply:
Since `gsutil` only supports short one, let's keep the consistence. @jonathanmetzman wdyt?

Reply:
Resolved and rejected this suggestion offline. Thanks.